interface/wx/filedlg.h

```diff
@@ -20,6 +20,11 @@ enum
 
 #define wxFD_DEFAULT_STYLE      wxFD_OPEN
 
+// set this system option to 1 in order to always show the filetypes popup in
```

Comment:
Does this actually work, i.e. does this expose this symbol in wxPython? Or do we need to use `///` or `/**` comments for it to be picked up?